lib/rubocop/cop/style/method_call_with_args_parentheses.rb

```diff
@@ -21,6 +21,12 @@ module Style
       #     to `true` allows the presence of parentheses in multi-line method
       #     calls.
       #
+      # 3. `AllowParenthesesInCamelCaseMethod` is `false` by default. This
```

Comment:
Given that the default behavior of this cop is to add parentheses, I would assume that this would be set to `true` by default.

Reply:
Actually, this option behaves the same as the other `AllowParenthesesIn*` options in that it takes effect when the style is `omit_parentheses` and provides exceptions to the rule. Thus, I believe it should be `false` by default (just like the other two `AllowParenthesesIn*` options) to preserve the current behavior.